content/basics/objects.md

```diff
@@ -22,22 +22,20 @@ Filters the array and returns a new array filtered by the
 callback function passed in. Callback should return true or false.
 
 ```js
-
```

Comment:
Not sure how much value is added by having these docs? Could you just present the MDN pages and have some explanations readily available instead?